test/cmdlineTests.sh

```diff
@@ -94,6 +94,56 @@ printTask "Testing unknown options..."
     fi
 )
 
+# General helper function for testing SOLC behaviour, based on file name, compile opts, exit code, stdout and stderr.
+# An failure is expected.
+test_solc_file_input_failures() {
+    local filename="${1}"
+    local solc_args="${2}"
+    local stdout_expected="${3}"
+    local stderr_expected="${4}"
+    local stdout_path=`mktemp`
+    local stderr_path=`mktemp`
+
+    set +e
+    "$SOLC" "${filename}" ${solc_args} 1>$stdout_path 2>$stderr_path
+    exitCode=$?
```

Comment:
Do you think it may be better to read the contents of ``$stdout_path`` and ``$stderr_path`` to some variables here already and remove the temporary files right away and before the ``set -e``? I always find it annoying if scripts clutter ``/tmp``. You're script shouldn't do that, but with ``set -e`` one can never be sure ;-). And it would save a few lines below.

And in general: do we need to distinguish stderr and stdout here? The other tests in this file don't...